### DISCUSSION TITLE
BT-53 is missing

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#891] [Q&A] [answered]
A BT-53 "Options" is mentioned in the Regulation Annex.
![grafik](https://github.com/OP-TED/eForms-SDK/assets/144460224/de552142-78e1-4e61-8034-fe926a8eae60)

However, I cannot find this BT either in fields.json or in the notices type definitions.

**Comment by YvesJo** [comment, marked as answer]
Hi, 
You may find a short explanation about the absence of BT-53 at https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html#pointlessDueToDesignSection
When BT-53 is set to true, then BT-54 is mandatory; and when set to false, it is forbidden. This may be simplified by specification of the Description of the Options when these exist and no Description when this does not.

**Reply by JanEmde** [reply]
I would ask you to scrutinise this again. On a technical level, this may be an simplification. However, it is not particularly intuitive for the employees of the awarding authorities:
With the current solution, the group starts with a field called "Description of the options".

It also contradicts the approach of the newer SDKs that many fields only appear when certain preconditions are met.

**Reply by YvesJo** [reply]
Hi,
The existence of many fields is conditioned, should it be for business reasons or just data consistency.
Rules were not active from the start as this was too much of a challenge and conditional rules have been progressively added to the SDK starting version 1.8.
When BT-53 would exist, then BT-54 existence would be conditioned on BT-53 value, however there is no condition expressed on BT-53 existence; therefore, there is no condition on BT-54 (which is taking over the condition that would have existed on BT-53 if any would have existed).

**Reply by JanEmde** [reply]
Nevertheless, I believe that this is counter-intuitive for the awarding organisations. Instead of first dealing with one field (options: yes/no), I get 3 fields whose content I don't immediately understand because the original question is not even displayed.

**Reply by YvesJo** [reply]
Associated to the 2 Business Terms (BT-53 and BT-54), there is only one field defined (BT-54-Lot) which is able to cover all the cases.
Please keep in mind that it is possible to tailor the label on the local implementation the way it better meets the user needs.
If label improvement would be the way to improve user experience, I recommend to submit this suggestion via the helpdesk.
